packages/react-reconciler/src/ReactFiberHooks.new.js

```diff
@@ -1498,12 +1504,24 @@ function rerenderDeferredValue<T>(
 
 function startTransition(setPending, config, callback) {
   const priorityLevel = getCurrentPriorityLevel();
+  const previousLanePriority = getCurrentUpdateLanePriority();
+  setCurrentUpdateLanePriority(
+    higherLanePriority(previousLanePriority, InputContinuousLanePriority),
+  );
   runWithPriority(
     priorityLevel < UserBlockingPriority ? UserBlockingPriority : priorityLevel,
     () => {
       setPending(true);
     },
   );
+
+  setCurrentUpdateLanePriority(
```

Comment:
This doesn't actually do anything, right? Since in `requestUpdateLane`, it will hit the SuspenseConfig branch and early return.

Reply:
Assuming I'm correct here, can you go ahead and delete this please? Along with a note explaining why, since we'll likely refactor `useTransition` soon.

Reply:
It does when there's no suspense config, e.g. [this test](https://github.com/facebook/react/blob/fe7163e73dadceda2655736d97cdd745d7abc8ea/packages/react-reconciler/src/__tests__/ReactTransition-test.js#L63).

Reply:
I think for this we should go back to using the NormalPriority/DefaultLanePriority like the scheduler uses below, otherwise we'll need to special case the warning for this case since, as you can see, they'll mismatch.

Reply:
The behavior then is "if you use startTransition without a suspense config then we use the default lane priority". We can update that later after we move off the scheduler priority if we want.

Reply:
Oh right

Reply:
I think in the future `startTransition` should set a default Suspense config object when one isn't provided explicitly. But let's punt on that for now.

Reply:
SGTM!